components/Receipt.js

```diff
@@ -338,7 +336,7 @@ export class Receipt extends React.Component {
 
                   <Box>
                     <H2 fontSize="16px" lineHeight="18px">
-                      {invoice.title || this.props.invoice.template?.title || (
```

Comment:
Shouldn't we deprecate https://github.com/opencollective/opencollective-api/blob/8571d308acb74cc70fa6108e09884717610c7a11/server/graphql/v1/queries.js#L182 if we're doing that?

Reply:
I've marked these `title` and `extraInfo` fields are deprecated in the relavant API PR,

https://github.com/opencollective/opencollective-api/blob/f86e7e6e31dab8a4f28aa0aede526827625b92ff/server/graphql/v1/types.js#L633

https://github.com/opencollective/opencollective-api/blob/f86e7e6e31dab8a4f28aa0aede526827625b92ff/server/graphql/v1/types.js#L642

Later once the API PR and this one is merged we should I think remove these fields as well from the API. 🤔